### DISCUSSION TITLE
How to start code-server in backgroud without systemctl inside docker?

### DISCUSSION THREAD

**Discussion by ct24p** [coder/code-server#6235] [Q&A]
not to run code server as docker container, wish to run a customized docker image (based on ubuntu 22.04 LTS) with code server installed. thanks

**Comment by code-asher** [comment]
You could start it the same way we do: https://github.com/coder/code-server/blob/main/ci/release-image/Dockerfile#L57

Or run it directly, something like:

```
ENTRYPOINT ["/usr/bin/code-server", "--bind-addr", "0.0.0.0:8080", "."]
```
